### PR TITLE
tweak: Triptch Platform direction

### PR DESCRIPTION
**Pull request by cmaddox5**
**Asana task**: [Name triptychs in Screenplay by direction](https://app.asana.com/0/1185117109217413/1205717671578677/f)

Added platform direction to Triptych screens. 

Screenshot 2023-11-16 at 12 35 12 PM

### CODE REVIEW

File: assets/js/components/Dashboard/ScreenDetail.tsx [hunk 1]
```diff
@@ -60,11 +61,23 @@ const ScreenCard = (props: ScreenDetailProps) => {
     }
   };
 
-  const getScreenLocation = isPaess
-    ? `/ ${getPaessRoute(paessRouteLetter)}`
-    : screens[0].location
-    ? `/ ${screens[0].location}`
-    : "";
+  const getTriptychPlatform = (directionId: number) =>
+    directionId === 0 ? "Southbound" : "Northbound";
```
Comment (jzimbel-mbta) [inline assets/js/components/Dashboard/ScreenDetail.tsx:65]: We'll need to revisit these labels if we ever put the triptych app on other lines (let alone the Orange Line... 😮‍💨) but this works for now.

**Review by jzimbel-mbta — APPROVED** [review]
👍

File: assets/js/components/Dashboard/ScreenDetail.tsx (hunk 1 above)
Comment (cmaddox5) [reply assets/js/components/Dashboard/ScreenDetail.tsx:65]: I thought about that too but decided to leave it with OL directions for now. I guess it wouldn't hurt to go ahead and do that now, right?

Comment (cmaddox5) [reply assets/js/components/Dashboard/ScreenDetail.tsx]: I went ahead and did it. Knowing how the rollout for OL triptychs went, we are likely going to forget about this tiny detail when adding our app to other routes (assuming that ever happens). I'd rather solve it now while it's fresh on my mind.

**Pull request merged**